services/blockchain-indexer/shared/dataService/business/pos/constants.js

```diff
@@ -13,12 +13,22 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
+const { Logger, LoggerConfig } = require('lisk-service-framework');
 const { requestConnector } = require('../../../utils/request');
+const config = require('../../../../config');
+
+LoggerConfig(config.log);
```

Comment:
Setting the `LoggerConfig` once in `app.js`should suffice. Please check if we can remove it.